microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1363,11 +1366,11 @@
                 <img class="diversity-banner" src='/static/img/{{ _(EN_US_KEY) }}/hero-home-4-xl.jpeg'>
                 <br/><br />
                 <p>
-                    {{ _('Your microbiome is a rainforest of diverse microbes dominated by microbes - some 40 trillion bacteria, weighing around 1 lb, live all over and in your body. The diversity of your microbiome, how similar it is to other peoples\', and the types of bacteria living inside you have all been linked to health, disease and lifestyle traits in numerous studies.') }}
+                    {{ _('There are 40 trillion microbial cells in our bodies. This collection of microbial cells is known as the human microbiome. With roughly 3lbs (1.4kgs) of them in our gut alone, we\'re on a mission to make critical discoveries about their role in our lives. Thank you for contributing to our research project and helping advance the science.') }}
```

Comment:
The weight is not supported, see https://www.nature.com/articles/s41564-023-01426-7, which suggests ~1lb

Reply:
Changed the verbiage